src/Uno.Templates/content/unoapp/MyExtensionsApp.1/Presentation/SecondPage.xaml

```diff
@@ -18,13 +22,26 @@
 				</AppBarButton>
 			</utu:NavigationBar.MainCommand>
 		</utu:NavigationBar>
+		<TextBlock Grid.Row="1" Text="{Binding Entity.Name}"
```

Comment:
To keep the template easier to manage, include the StackPanel in both cases and just put conditions around the Back Button itself

Reply:
Done. But i needed to add two IFs, to have the title of the page and the correct position of the Naviagation.